.github/workflows/check_notebooks.yml

```diff
@@ -0,0 +1,41 @@
+name: Check notebooks
+
+on:
```

Comment:
do we want this upon every pull_request?

Reply:
just for the record, we decided to move forward with running this manually and upon building docs upon release.